system-tests/src/test/java/org/eclipse/edc/samples/advanced/advanced00opentelemetry/Advanced00openTelemetryTest.java

```diff
@@ -0,0 +1,86 @@
+/*
+ *  Copyright (c) 2023 Mercedes-Benz Tech Innovation GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Mercedes-Benz Tech Innovation GmbH - Sample workflow test
+ *
+ */
+
+package org.eclipse.edc.samples.advanced.advanced00opentelemetry;
+
+import org.apache.http.HttpStatus;
+import org.eclipse.edc.connector.transfer.spi.types.TransferProcessStates;
+import org.eclipse.edc.junit.annotations.EndToEndTest;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.DockerComposeContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+import static org.eclipse.edc.samples.transfer.FileTransferCommon.getFileContentFromRelativePath;
+import static org.eclipse.edc.samples.transfer.FileTransferCommon.getFileFromRelativePath;
+import static org.eclipse.edc.samples.transfer.TransferUtil.checkTransferStatus;
+import static org.eclipse.edc.samples.transfer.TransferUtil.startTransfer;
+import static org.eclipse.edc.samples.transfer.transfer00prerequisites.PrerequisitesCommon.runPrerequisites;
+import static org.eclipse.edc.samples.transfer.transfer01negotiation.NegotiationCommon.createAsset;
+import static org.eclipse.edc.samples.transfer.transfer01negotiation.NegotiationCommon.createContractDefinition;
+import static org.eclipse.edc.samples.transfer.transfer01negotiation.NegotiationCommon.createPolicy;
+import static org.eclipse.edc.samples.transfer.transfer01negotiation.NegotiationCommon.getContractAgreementId;
+import static org.eclipse.edc.samples.transfer.transfer01negotiation.NegotiationCommon.negotiateContract;
```

Comment:
if this samples need code from the `transfer` type tests, I'd say that these common class should stay in an higher level package, like `org.eclipse.edc.samples.common`